src/components/AvatarWithDisplayName.js

```diff
@@ -91,7 +91,7 @@ function AvatarWithDisplayName(props) {
                         {!_.isEmpty(parentNavigationSubtitleData) && (
                             <ParentNavigationSubtitle
                                 parentNavigationSubtitleData={parentNavigationSubtitleData}
-                                report={props.report}
+                                parentReportID={props.report.parentReportID || ''}
```

Comment:
Can you please remove the fallback value. `ParentNavigationSubtitle` `parentReportID` default value is already an empty string. (samething for HeaderView)

Reply:
`props.report.parentReportID` `parentReportID` is optional and may not exist on the report object

Reply:
https://github.com/Expensify/App/blob/b361d2bb2725b14b8b0ba46e4e32fc583f4ce81e/src/components/ParentNavigationSubtitle.js#L29